widget/class-widget.php

```diff
@@ -61,6 +64,12 @@ public function widget( $args, $instance ) {
             )
          );
 
+         // Set default title if empty
+         $instance['title'] = empty( $instance['title'] ) ? 'Flexible Volunteer Opportunities' : $instance['title'];
```

Comment:
Could we default it to just 'Volunteer Opportunities' and add a line in the admin that states 'if left blank title will default to Volunteer Opportunities'?